aws-datastore/src/main/java/com/amplifyframework/datastore/storage/sqlite/adapter/SQLPredicate.java

```diff
@@ -187,6 +187,21 @@ private StringBuilder parsePredicateOperation(QueryPredicateOperation<?> operati
                         .append("?");
             case EQUAL:
             case NOT_EQUAL:
+                Object operatorValue = getOperatorValue(op);
+                if (operatorValue != null) {
+                    addBinding(operatorValue);
+                    return builder.append(column)
+                        .append(SqlKeyword.DELIMITER)
+                        .append(SqlKeyword.fromQueryOperator(op.type()))
+                        .append(SqlKeyword.DELIMITER)
+                        .append("?");
```

Comment:
Since there is no "break" statement, you don't need to repeat this code here and just check for special case using if statement.  Codepath the does not fall into if statement will continue executing lines at 209.

Reply:
I like it!!! Making that change right now